app/views/application/_success_flash.html.erb

```diff
@@ -0,0 +1,7 @@
+<% flash.each do |name, msg| %>
+  <div class="flash-archive">
+    <svg class="moj-banner__icon" color="green" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+      <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z"></path></svg>
+    <%= msg.to_s %>
+  </div>
+<% end %>
```

Comment:
There is already a [flash partial][1] can we reuse it?

[1]: https://github.com/unboxed/bops/blob/master/app/views/application/_flashes.html.erb

Reply:
The styling is completely different for this particular message - it needs to look like the MoJ banner. I thought it would be not ideal to have two completely different sets of css in the same partial, but if you can think of a way to do it then please suggest